herbert_core/utilities/classes/@IX_dataset/IX_dataset.m

```diff
@@ -1,4 +1,4 @@
-classdef IX_dataset < serializable & data_op_interface
+classdef IX_dataset < data_op_interface & serializable
```

Comment:
to be fair, I do not know. serializable looks like higher order inheritance but how to express this on interface -- unknown thing. Here I've decided that higher order inheritance should be furter from the class then the inheritance, more specific to the class, but it may be interpreted other way around. 

I've tried this tought here -- I do not see the difference.

Reply:
ok let's not make any more changes like this if it makes no difference